### PR TITLE
Move forked header CSS to theme

### PR DESCRIPTION
**Pull request by caesar2164**
@stvstnfrd matching theme changes to move forked SASS

### CODE REVIEW

**Comment by caesar2164** [comment]
matches https://github.com/Stanford-Online/edx-platform/pull/770

File: cme/lms/static/sass/_overrides.scss [hunk 1]
```diff
@@ -89,19 +89,22 @@
...
         h2.course-header {
+            margin: 9px 0 0 0;
             float: left;
             font-family: $sans-serif;
             font-size: 0.9em;
             font-weight: 600;
             color: #515553;
             letter-spacing: 0;
-            margin: 0;
-            margin-top: 9px;
```
Comment (stvstnfrd) [inline cme/lms/static/sass/_overrides.scss:104]: Why this change now?

Comment (caesar2164) [reply cme/lms/static/sass/_overrides.scss:104]: Cause the way we did it before was just stupid?
Also I moved it to a more natural position for `margin`

Comment (stvstnfrd) [reply cme/lms/static/sass/_overrides.scss:104]: Ahhhhh, I got it now.

**Review by stvstnfrd — APPROVED** [review]
